docs/releases/2.2.txt

```diff
@@ -140,6 +140,9 @@ Database backends
 
 * Added result streaming for :meth:`.QuerySet.iterator` on SQLite.
 
+* The Oracle backend client is modified so that the :djadmin:`dbshell`
+  management command will be called wrapped with ``rlwrap`` if available.
```

Comment:
`called wrapped`

Reply:
Hi Simon, but it is `called wrapped` :smile:  Am I missing sth? :thinking:

Reply:
I could be missing something as I'm not a native speaker but _management command will be called wrapped with_ sounded funny to be. I would have expected _called_ to be dropped but I'll let someone else chime in.

Reply:
Agreed, I will drop `called` :+1: .